vxdev/update-code.sh

```diff
@@ -49,6 +53,21 @@ elif [[ $BRANCH == 'stable' ]]; then
 	cd vxsuite
 	./script/setup-dev
 	cd ..
+elif [[ $BRANCH == 'custom' ]]; then
+	read -p "Enter the branch name: " BRANCH_NAME
+	cd vxsuite
+	git checkout main
+	git pull
+	while [ !`git branch -r --list origin/$BRANCH_NAME` ]
+	do
+		read -p "Invalid Branch Name. Try again: " BRANCH_NAME
+	done
```

Comment:
awesome, my only nudge might be to make the prompt in the loop a bit clearer about what you're asking for. Instead of "Try again," maybe "Please enter a valid branch name:" or something